src/main/java/uk/ac/ebi/ega/egacryptor/cryptography/pgp/PGPUtils.java

```diff
@@ -88,7 +92,7 @@ public static PGPPublicKey readPublicKey(final InputStream inputStream) throws I
 
     public static PGPEncryptedDataGenerator getEncryptedGenerator(final PGPPublicKey pgpPublicKey) {
         final PGPEncryptedDataGenerator pgpEncryptedDataGenerator = new PGPEncryptedDataGenerator(
-                new JcePGPDataEncryptorBuilder(PGPEncryptedData.CAST5).
+                new JcePGPDataEncryptorBuilder(SymmetricKeyAlgorithmTags.CAST5).
```

Comment:
Sonar told me that I should use the variable from the parent class, not from the subclass.